debian/get-orig-source.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+set -eux -o pipefail
+
```

Comment:
I think it's worth checking if the `debian/` directory exists here and error out if it doesn't.

Reply:
Judging from the contents of lines 13/14, this script is expecting to be executed as `./debian/get-orig-source.sh`, right?  In this case, I think it's worth making sure that it is indeed being executed inside the source tree but outside the `debian/` directory, otherwise it will fail.

Reply:
Got it now, this is to make sure that the script is called from the root of the source package. +1.